src/Sylius/Behat/Context/Ui/Admin/ManagingPaymentMethodsContext.php

```diff
@@ -0,0 +1,372 @@
+     */
+    public function iChooseGateway($gatewayName)
+    {
+        $currentPage = $this->currentPageResolver->getCurrentPageWithForm($this->createPage, $this->updatePage);
+
+        $currentPage->chooseGateway($gatewayName);
+    }
+
+    /**
+     * @Then I should be notified that it has been successfully edited
+     */
+    public function iShouldBeNotifiedAboutSuccessfulEdition()
+    {
+        $this->notificationChecker->checkEditionNotification(self::RESOURCE_NAME);
+    }
+
+    /**
+     * @Then this payment method :element should be :value
+     */
+    public function thisPaymentMethodElementShouldBe($element, $value)
+    {
+        Assert::true(
+            $this->updatePage->hasResourceValues([$element => $value]),
+            sprintf('Payment method should have %s %s', $value, $element)
+        );
+    }
+
+    /**
+     * @Given I want to create a new payment method
+     */
+    public function iWantToCreateANewPaymentMethod()
+    {
+        $this->createPage->open();
+    }
+
+    /**
+     * @When I specify its code as :code
+     * @When I do not specify its code
+     */
+    public function iSpecifyItsCodeAs($code = null)
+    {
+        $this->createPage->specifyCode($code);
+    }
+
+    /**
+     * @When I describe it as :description in :language
+     */
+    public function iDescribeItAsIn($description, $language)
+    {
+        $this->createPage->describeIt($description, $language);
+    }
+
+    /**
+     * @When I add it
+     * @When I try to add it
+     */
+    public function iAddIt()
+    {
+        $this->createPage->create();
+    }
+
+    /**
+     * @Then I should be notified that it has been successfully created
+     */
+    public function iShouldBeNotifiedItHasBeenSuccessfulCreation()
+    {
+        $this->notificationChecker->checkCreationNotification(self::RESOURCE_NAME);
+    }
+
+    /**
+     * @Then the payment method :paymentMethodName should appear in the registry
+     * @Then the payment method :paymentMethodName should be in the registry
+     */
+    public function thePaymentMethodShouldAppearInTheRegistry($paymentMethodName)
+    {
+        $this->indexPage->open();
+
+        Assert::true(
+            $this->indexPage->isResourceOnPage(['name' => $paymentMethodName]),
+            sprintf('Payment method with name %s has not been found.', $paymentMethodName)
+        );
+    }
+
+    /**
+     * @When I want to browse payment methods
+     */
+    public function iWantToBrowsePaymentMethods()
+    {
+        $this->indexPage->open();
+    }
+
+    /**
+     * @Then I should see :amount payment methods in the list
+     */
+    public function iShouldSeePaymentMethodsInTheList($amount)
+    {
+        $foundRows = $this->indexPage->countItems();
+
+        Assert::eq(
+            ((int) $amount),
+            $foundRows,
+            sprintf('%s rows with payment methods should appear on page, %s rows has been found', $amount, $foundRows)
```

Comment:
`'%2$s rows with payment methods should appear on page, %s rows has been found'` no need for `sprintf`, this string will be enough